File: week03-exam-web/src/main/webapp/js/service/goodsService.js
/**
 * 
 */
app.service('goodsService',function($http){
	this.getList=function(){
		return $http.get('../goods/list.do');
	}
	
	this.out=function(id){
		return $http.get('../goods/out.do?id='+id);
	}
	
	this.back=function(id){
		return $http.get('../goods/back.do?id='+id);
	}
	
	this.tegezer=function(){
		return $http.get('../goods/tegezer.do');
	}

	this.getSolrList=function(){
		return $http.get('../goods/getSolrList.do');
	}
	
	this.deleteId=function(id){
		return $http.get('../goods/delete.do?id='+id);
	}
	
	this.cx=function(name){
		return $http.get('../goods/cx.do?name='+name);
	}
})

File: week03-exam-interface/src/main/java/com/week03/exam/service/GoodsService.java
package com.week03.exam.service;

import java.util.List;

import com.week03.exam.pojo.Goods;

public interface GoodsService {

	List<Goods> list();

	void out(int id);

	void back(int id);

	List<Goods> findByZt();

	void tegezer();

	List<Goods> getSolrList();

	void delete(int id);

	/*void cx(String name);*/

}
